### DISCUSSION TITLE
More advanced callout card

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3394] [Help: Figma] [answered]
hey, we want to recreate following component using Paste DS (_blue card - Create Tracking Plan_), any ideas how to achieve similar effect? Callout component seems the closest to what we want to have, however there are no buttons as a variant, besides buttons there is no status for this case, sort of "idea". thanks in advance!

<img width="782" alt="Zrzut ekranu 2023-08-9 o 13 12 39" src="https://github.com/twilio-labs/paste/assets/140410476/6f1712c4-035c-461c-bff5-4305e4799ef6">

**Comment by loreina** [comment, marked as answer]
Hi @marektonik,

The Callout is composable and you can nest any items inside, including a Button. We don't have an "idea" status, though I think a [Card](https://paste.twilio.design/components/card) or [neutral Callout](https://paste.twilio.design/components/callout#neutral-callout) might work here depending on the importance of the content being conveyed.

**Reply by ceyerkes** [reply]
@marektonik - We should collaborate with the design POC and re-think these more complicated cards altogether. There is a lot going on in this card, and I'm not sure that the blue help info (help and settings cards should just use Paste's Card component) within the card is the right call, but I'd like to chat with the designer first. 

Let's add this to our list, but in the meantime, can you please take a stab at creating this with current Paste components? I'd love more context, but I'm thinking the blue box stuff could probably be an Anchor.

**Reply by marektonik** [reply]
@loreina how to put items inside e.g. a Button? There is no such variant with custom slot

**Reply by loreina** [reply]
Hi @marektonik, what custom content are you trying to put inside a Button?

**Reply by marektonik** [reply]
@loreina I wanted to put a Button inside a Callout. By custom slot I meant custom slot for callout body, sth more adavanced than regular text or list.

**Reply by loreina** [reply]
@marektonik In code, Callout is composable so you can put a Button in there. We don't yet have a Callout slot variant in Figma. I'd echo my previous suggestion of looking into Card as an option depending on the use case and importance of the content being conveyed.

**Reply by marektonik** [reply]
@loreina  the thing is we don't have much space there :( A Card component has quite big paddings. Until we don't come up with better solution, can we just detach a Callout and put a Button there?

**Reply by SiTaggart** [reply]
If I'm not mistaken, you can adjust the padding in the Card in Figma, just like in Code, using the layout properties.

A card would probably work just fine here.
